.github/workflows/transcendence-app_check.yml

```diff
@@ -0,0 +1,29 @@
+name: transcendence-app check
+
+on:
+  push:
+    branches: [$default-branch]
+  pull_request:
+    types: [assigned, opened, synchronize, reopened]
```

Comment:
Should we also add this part to webapp_check.yml?

Reply:
yup, I copy-pasted only part of it :sweat_smile: